demos/bootstrap.js

```diff
@@ -0,0 +1,25 @@
+(function() {
+
+	// Find the script element
+	var scripts = document.getElementsByTagName( "script" );
+	var script = scripts[ scripts.length - 1 ];
+
+	// Read the modules
+	var modules = script.getAttribute( "data-modules" );
+	var demoScripts = script.getAttribute( "data-modules" );
+	var path = window.location.pathname;
+	var pathParts = path.split( "/" );
+	var pathLength = pathParts.length;
+	if ( modules ) {
```

Comment:
my personal thought was in the case of demos this was fine we don't need to handle nearly the variety of situations in demos just compare the bootstrap file complexity